combat-trainer.lic

```diff
@@ -1487,7 +1487,7 @@ class SpellProcess
 
     if Time.now - @perc_health_timer > 30 && @empath_spells['FOC'] || @empath_spells['HEAL']
       @perc_health_timer = Time.now
-      @wounds = check_perc_health
+      @wounds = DRCH.perceive_health['wounds']
```

Comment:
Updates to new method name and data structure of return type.

Reply:
This looks like a breaking change to me.

If so, what I suggest is one PR that adds in the new `perceive_health` function, and then another that removes the old `check_perc_health` function.

Reply:
That makes sense, and makes this PR smaller so that it focuses more on the new logic and not how existing scripts (outside of healme/common-healing) adapt to the function changes. I'll split it up.